docs/spec/dataclasses.rst

```diff
@@ -365,6 +365,12 @@ positional and may use other names.
 * ``alias`` is an optional str parameter that provides an alternative
   name for the field. This alternative name is used in the synthesized
   ``__init__`` method.
+* ``converter`` is an optional parameter that specifies a single-argument
```

Comment:
Should say "single-parameter", not "single-argument". Callables have parameters; argument are passed to the callable.

Reply:
```suggestion
* ``converter`` is an optional parameter that specifies a single-parameter
```